.github/workflows/e2e-short.yaml

```diff
@@ -8,6 +8,23 @@ jobs:
   e2e:
     runs-on: ubuntu-latest
     steps:
+      - name: Cleanup space
```

Comment:
this looks duplicate to me, can you please rebase?

Reply:
Fixed, waiting for e2e tests to pass.